internal/cephfs/clone.go

```diff
@@ -94,6 +96,9 @@ func createCloneFromSubvolume(ctx context.Context, volID, cloneID volumeID, volO
 	case cephFSCloneInprogress:
 		util.ErrorLog(ctx, "clone is in progress for %v", cloneID)
 		return ErrCloneInProgress
+	case cephFSClonePending:
+		util.ErrorLog(ctx, "clone is pending for %v", cloneID)
```

Comment:
@Yuggupta27 may be in next PR convert this Error logs to Debug logs

Reply:
Sure Madhu... Will do.